app/views/admin/authors/_author.html.erb

```diff
@@ -1,5 +1,9 @@
+<% @index ||= @pagy.offset %>
 <%= table_body do %>
   <%= table_row do %>
+    <%= table_column do %>
+      <%= @index += 1 %>
```

Comment:
Thử sử dụng counter trong collection nhé
https://www.mintbit.com/blog/rails-counter-available-in-collection-partials

Reply:
Dạ sử dụng này chuyển trang bị quay lại 1 ạ
![image](https://github.com/user-attachments/assets/91dc95e6-8b4e-4838-9fd5-f099d431fdd1)
![image](https://github.com/user-attachments/assets/5b7bafcf-99ba-4946-9f02-c0cff803bd05)